collectors/python.d.plugin/openldap/openldap.chart.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Description: openldap netdata python.d module
+# Author: Manolis Kartsonakis (ekartsonakis)
+# SPDX-License-Identifier: GPL-3.0+
+
+try: 
+     import ldap 
+     HAS_LDAP = True 
+except ImportError: 
+     HAS_LDAP = False
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+# default module values (can be overridden per job in `config`)
+update_every = 10
+priority = 60000
+retries = 60
+
+ORDER = [
+    'total_connections', 'bytes_sent',
+    'operations', 'referrals_sent',
+    'entries_sent', 'ldap_operations', 'waiters'
+]
+
+CHARTS = {
+  
+     'total_connections': {
+        'options': [None, 'Total Connections', 'connections', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['total_connections', None, 'incremental']
+        ]
+    },
+  
+    'bytes_sent': {
+        'options': [None, 'Bytes Statistics', 'Bytes', 'ldap', 'monitorCounter', 'line'],
+        'lines': [
+            ['bytes_sent', None, 'incremental']
```

Comment:
1. Please add dimension name.

```python
['bytes_sent', 'sent', 'incremental']
```

^^ please do so for all charts


2. convert bytes to KB

```pyhon
['bytes_sent', None, 'incremental', 1, 1024]
```

Reply:
done

Reply:
done